test/ApiClient.test.ts

```diff
@@ -296,25 +282,7 @@ describe('Subscribe', () => {
   })
 
   it('should resubscribe on completion', async () => {
-    let called = 0
-    const subscribeMock = jest
-      .spyOn(MessageApi, 'Subscribe')
-      .mockImplementation(
-        async (
-          req: messageApi.SubscribeRequest,
-          cb: NotifyStreamEntityArrival<messageApi.Envelope> | undefined
-        ): Promise<void> => {
-          called++
-          if (called == 1) {
```

Comment:
same question here: was the previous "return" behavior intended to differ from the normal `createSubscribeMock` behavior? @snormore worth getting your eyes before merge